src/main/webapp/app/shared/util/course.util.ts

```
@@ -4,8 +4,12 @@ import { Course } from 'app/entities/course.model';
  * Sorts an array of Course objects alphabetically by their title.
  *
  * @param {Course[]} courses - The array of Course objects to be sorted.
+ * @param {boolean} isSortAscending - Sort direction, it is ascending by default.
  * @returns {Course[]} The sorted array of Course objects.
  */
-export function sortCourses(courses: Course[]): Course[] {
-    return courses.sort((courseA, courseB) => (courseA.title ?? '').localeCompare(courseB.title ?? ''));
+export function sortCourses(courses: Course[], isSortAscending: boolean = true): Course[] {
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)